locale/core.yml

```diff
@@ -19,12 +19,15 @@ core:
       colored_header_label: Colored Header
       colors_heading: Colors
       colors_text: "Choose two colors to theme your forum with. The first will be used as a highlight color, while the second will be used to style background elements."
+      custom_footer_heading: Custom Footer
+      custom_footer_text: "Add HTML to be displayed at the very bottom of the page."
```

Comment:
Please order the keys alphabetically. ;)

Reply:
sure, thing. will change it asap

Reply:
Almost. You forgot these ones... ;)

Reply:
I'm sorry. But can you help me figure what's i miss?

I've tried to sort it using Sublime's sort function. But nothing happen :(

Reply:
Uhm, must have been a bug in my brain. :confused:

Reply:
no worries 😄